include/elliptic/AffinePointAsBinary.h

```diff
@@ -0,0 +1,63 @@
+#ifndef __CRYPTID_AFFINEPOINT_AS_BINARY_H
+#define __CRYPTID_AFFINEPOINT_AS_BINARY_H
+
+#include "elliptic/AffinePoint.h"
+
+/**
+ * Represents an affine point with binaries of coordinates.
+ */
+typedef struct AffinePointAsBinary
+{
+    /**
+     * The binary representation of the \f$x\f$ coordinate.
+     */
+    char *x;
```

Comment:
Not a major thing, but I think, that `void *` might be a better choice here. In my opinion, that would carry the intent more faithfully (that, this is binary data). What do you think?

Another argument in favor of `void *` is the fact, that both `mpz_export` and `mpz_import` work with this type.

The same applies for other such fields.

Reply:
Well yes, that would fit well, but in my opinion, using `void *` is not too expressive. As we know what we want to store why would we try to stay with the generic property?

Reply:
Sure, we know it, but our clients don't. What is more, they should not even know it, since the format of these fields in an implementation detail. We just happen to handle some binary data here, nothing special to see for clients.